.github/workflows/rust.yml

```
@@ -75,42 +76,32 @@ jobs:
-  #       run: |
-  #         cd ./ibc-tests
-  #         npm i
-  #         npm test
-
   build-schemas:
     runs-on: ubuntu-latest
     name: Actions - build schemas
     needs: contract-tests
+    if: github.event_name == 'pull_request'
     steps:
       - uses: actions/checkout@v2
```

actionlint

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)